### ISSUE TITLE
Lasair Docs

### ISSUE THREAD

**Issue by thespacedoctor** [lsst-uk/lasair_api#6]
I propose placing the Lasair docs in this repo. If I place the docs here, I can also expose the methods for the Lasair API wrapper found here.

How does that sound Lasair dev team?

**Comment by RoyWilliams** [comment]
Sounds great. Will you merge whats now in the github with https://lasair-iris.roe.ac.uk/api?

**Comment by thespacedoctor** [comment]
I think so. I have the readthedocs template all set up (in this repo, but only on my mac so far) and have started to port various docs into it. This is the most public-facing repo of all the lasair code repos so makes sense to add the docs here. I'll report back here once I have something to show.

**Comment by thespacedoctor** [comment]
I've now ported all plain-text docs out of the Lasair webpages into sphinx (documentation engine) & readthedocs (public platform that hosts sphinx-driven docs).

I have carefully thought through the structure of the documentation (page & section naming etc), but have simply copy and pasted documentation text from the old web pages into appropriate sections. I/we can update the documentation during/after web page refresh.

I have added 'todos' in the docs where work on the documentation needs to be done ... probably for my future self (see todo list in the sidebar).

<https://lasair.readthedocs.io/en/feature-adding_docs/>

The goal of this stage of the web page refresh was to move documentation out of the Lasair webapp into a suitable home. This is now done, apart from the API documentation which I shall do at a later stage.
